### ISSUE TITLE
ExecutionTest::ATOProgOffset is failing (shader model 6.7)

### ISSUE THREAD

**Issue by hekota** [microsoft/DirectXShaderCompiler#5835]
**Description**
ExecutionTest::ATOProgOffset is failing

**Steps to Reproduce**
`hcttest exec-filter *ATOProgOffset -adapter NVidia*`

The failure first showed up with commit 1dac9d8f518b429c3b9f95aad9fcb58d1e0d4591.

**Actual Behavior**

```
StartGroup: ExecutionTest::ATOProgOffset
Using Adapter:NVIDIA GeForce GTX 1660 Ti
Error: Verify: AreEqual(pPixels[2 * 4 * ix + 4], cmp) - Values (107942, 107958) [File: D:\dxc2\tools\clang\unittests\HLSLExec\ExecutionTest.cpp, Function: VerifyProgOffsetResults, Line: 4440]
EndGroup: ExecutionTest::ATOProgOffset [Failed]
```

**Environment**
- DXC version 769b504be4748931254036e3250c1daad5a1991a
- Host Operating System Windows

**Comment by tex3d** [comment]
Yes, that would be because of this, most likely:

https://github.com/microsoft/DirectXShaderCompiler/commit/1dac9d8f518b429c3b9f95aad9fcb58d1e0d4591#r129548831

**Comment by hekota** [comment]
Fixed by #5847.
